src/components/Metadata.vue

```diff
@@ -93,8 +93,18 @@ export default {
         margin-bottom: $block-margin;
         text-align: left;
 
+        h4 {
+            font-size: 1.2em;
+        }
+
         .row {
```

Comment:
Something weird is going on with the metadata after merging:

![grafik](https://user-images.githubusercontent.com/8262166/159355943-bc35b14e-0b4e-4000-9b85-55e98e76b2df.png)

Reply:
Fixed, there was a margin: 0 missing